### PR TITLE
Add Serverless

### PR DESCRIPTION
**Pull request by anaibol**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please read and adhere to the [contributing guidelines](../blob/master/contributing.md).

**Comment by anaibol** [comment]
Hi, I checked that but I don't see whats exactly missing, could you help me please?

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/blob/master/contributing.md#the-pull-request-should-have-a-useful-title-pull-requests-with-update-readmemd-as-title-will-be-closed-right-away-because-im-so-tired-of-nobody-reading-this-document-please-carefully-read-everything-in-adding-to-this-list

**Comment by anaibol** [comment]
Done!

**Comment by sindresorhus** [comment]
Some feedback:

- You have yet to read and follow: https://github.com/sindresorhus/awesome/blob/master/create-list.md
- Be consistent about ending all descriptions in a `.`
- Add a description to all items.
- Maybe right align the logo. Example: https://github.com/sindresorhus/awesome-nodejs/commit/d39ee6acf2c9cc7e5cf4c20664ab173af7de06eb

**Comment by anaibol** [comment]
Cool. Thanks! I Will work on that.

**Comment by sindresorhus** [comment]
Any updates?

**Comment by anaibol** [comment]
Hi, yes, I aligned right the logo and fixed all descriptions to end with a "."

**Comment by davisonio** [comment]
I've also noticed here's also a similar list: https://github.com/JustServerless/awesome-serverless. Looks like this one is on general server less resources whereas the one linked above is on the server less project itself.

You could simplify a few link titles `http://nobackend.org/` -> `nobackend.org` and `https://github.com/auth0/webtask-scripts/tree/master/stripe`.

Contributing guidelines?

**Comment by nikgraf** [comment]
Serverless is already listed under backend: https://github.com/sindresorhus/awesome#back-end-development

**Comment by anaibol** [comment]
"Serverless" is a concept that the "Serverless" framework is using, but these are two different things!

**Comment by anaibol** [comment]
@davisonio, added contiburting guidelines and fixed some links

**Comment by davisonio** [comment]
The descriptions under "push notifications" should have a `-` separating the link from the item. There's also still a few inconsistencies in the descriptions, some are non-existent and some are missing a `.` at the end.

Can you also fix the merge conflicts?

File: readme.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Charting](https://github.com/zingchart/awesome-charting)
 - [Ionic Framework 2](https://github.com/candelibas/awesome-ionic2)
 - [Chrome DevTools](https://github.com/ChromeDevTools/awesome-chrome-devtools)
-
```
Comment (sindresorhus) [inline readme.md]: Don't remove the newline

File: readme.md [hunk 2]
```diff
@@ -173,7 +173,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
...
+- [Serverless](https://github.com/anaibol/awesome-serverless/)
```
Comment (sindresorhus) [inline readme.md]: Drop the trailing `/ `

**Pull request closed without merging**